services/events.service.js

```diff
@@ -245,22 +234,22 @@ class EventsService extends BaseService {
     let response
 
     try {
-      const dataSnapshot = await this.collection.doc(id).get();
+      const dataSnapshot = await this.collection.doc(id).get()
 
-      event = dataSnapshot.exists ? dataSnapshot.data() : null;
+      event = dataSnapshot.exists ? dataSnapshot.data() : null
 
       if (!event) {
-        throw new Error({message: `The event id ${id} was not found`})
+        throw new Error({ message: `The event id ${id} was not found` })
       }
 
-      event.id = id;
+      event.id = id
 
       const imageIndex = event.images.findIndex(image => {
```

Comment:
Tambien podemos usar destructuring aqui 
```suggestion
      const imageIndex = event.images.findIndex({ id }=> {
```

Reply:
Se realizo la desestructuración